### DISCUSSION TITLE
Budibase Init - Error: self signed certificate in certificate chain

### DISCUSSION THREAD

**Discussion by StefanJenni** [Budibase/budibase#3653] [Help] [answered]
Hi
I have installed Budibase on a Windows 2019 server. But when I run the command
**budi hosting --init**
I get the following error message:
_Error - Error: self signed certificate in certificate chain_
Could somebody help me?
Kind regards,
Stefan

**Comment by StefanJenni** [comment, marked as answer]
Hi
We found out that our SSL inspection caused the problem with the "...certificate chain". After disabling SSL inspection, the init-process was successfull.
Stefan

**Reply by mjashanks** [reply]
Hey @StefanJenni - apologies that we didn't get back to you, missed this one. Feel free to bump it up in future!

Thank you for reporting back the solution to your problem - very much appreciated. This will help others :)
